server/src/rest/macros.rs

```diff
@@ -1,4 +1,4 @@
 //! log macro's for REST logging
 
 use lib_common::log_macros;
-log_macros!("rest");
+log_macros!("rest", "app::rest_requests");
```

Comment:
This shouldn't be changed, log4rs is configured to log all `app::rest` targets to the `rest_requests` appender. The target itself is not called `app::rest_requests` though.

Reply:
https://github.com/aetheric-oss/svc-assets/pull/27/commits/7b6a3357ee5f40abfaf138c6530cc37478e0f4ae fixed